comment/tests/test_version.py

```diff
@@ -0,0 +1,19 @@
+import os
+from unittest.mock import patch, mock_open
+
+from django.test import TestCase
+
+from comment import _get_version
+
+
+class TestGetVersion(TestCase):
+    _parent_project_dir = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+    version_file = os.path.join(_parent_project_dir, 'VERSION')
```

Comment:
Since you are mocking theversion value below why would you read the real value from the file? the variable `version_file` is unused.
In my opinion it would be more realistic if you assert the `_get_version()` with the actual value in `version_file`.

Reply:
No, the actual value will keep changing. So the test will also require a change every time there's a change of version.

Regarding the variables, I was using them for debugging something. Forgot to remove. will remove them if everything else looks good to you.

Reply:
> No, the actual value will keep changing. So the test will also require a change every time there's a change of version.

the value is read from the VERSION file so if you compare `version_file.read().strip()` with `_get_version()` will do the trick and will not require to change anything when the version is incremented.

However, feel free to use either way!

Reply:
> the value is read from the VERSION file so if you compare version_file.read().strip() with _get_version() will do the trick

It will not be in every case. Suppose I mock the contents to be `v2.0.0`. The function `_get_version()` would return `2.0.0` which would not be the same as contents of the file.